src/_cffi_src/openssl/pkcs7.py

```diff
@@ -10,7 +10,40 @@
 
 TYPES = """
 typedef struct {
+    Cryptography_STACK_OF_X509 *cert;       /* [ 0 ] */
+    Cryptography_STACK_OF_X509_CRL *crl;    /* [ 1 ] */
```

Comment:
Ok, I saw some comments in x509v3.py, so I wasn't sure whether to leave them in or not.